dependencies.yaml

```diff
@@ -302,11 +302,6 @@ dependencies:
       - output_types: [conda, pyproject, requirements]
         packages:
           - cython>=3.0.0
-  python_run_libwholegraph:
```

Comment:
Looks like this was added in #182, probably copying patterns from `libcugraph`.

It looks "unused" here because it wasn't wired up to populate `python/libwholegraph/pyproject.toml`. There is currently an import of `rapids_logger` in this project... but I believe that's also unnecessary and also probably copied from another project.

This import and any uses of it should be removed too:

https://github.com/rapidsai/cugraph-gnn/blob/e1ec28870116ec13d52bcfaac0eba1d5037f03e1/python/libwholegraph/libwholegraph/load.py#L51

As should all of this stuff, I think:

https://github.com/rapidsai/cugraph-gnn/blob/e1ec28870116ec13d52bcfaac0eba1d5037f03e1/dependencies.yaml#L473

https://github.com/rapidsai/cugraph-gnn/blob/e1ec28870116ec13d52bcfaac0eba1d5037f03e1/dependencies.yaml#L108